src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -200,9 +200,9 @@ private protected CalibratorTransformer(IHostEnvironment env, ModelLoadContext c
             }
         }
 
-        string ISingleFeaturePredictionTransformer<TICalibrator>.FeatureColumnName => DefaultColumnNames.Score;
+        public string FeatureColumnName => DefaultColumnNames.Score;
```

Comment:
I think I would leave these as explicitly implementing the interface. You can just add another one for the new interface:

```C#
        string ISingleFeaturePredictionTransformer.FeatureColumnName => DefaultColumnNames.Score;
```

Reply:
Fixed.